components/index.scss

```diff
@@ -1,3 +1,6 @@
+@import '../bower_components/flexboxgrid/dist/flexboxgrid';
```

Comment:
It's not part of components anymore. Probably we didn't resolve merge conflicts correctly.